src/Jackett.Common/Indexers/Anidex.cs

```diff
@@ -0,0 +1,387 @@
+                {"size", "size"},
+                {"filename", "title"}
+            }) { Name = "Sort by", Value = "upload_timestamp" };
+            configData.AddDynamic("sortrequestedfromsite", sortBySelect);
+
+            SelectItem orderSelect = new SelectItem(new Dictionary<string, string>()
+            {
+                {"desc", "Descending"},
+                {"asc", "Ascending"}
+            })
+            { Name = "Order", Value = "desc" };
+            configData.AddDynamic("orderrequestedfromsite", orderSelect);
+        }
+
+        public string SortBy
+        {
+            get
+            {
+                return ((SelectItem)this.configData.GetDynamic("sortrequestedfromsite")).Value;
+            }
+        }
+
+        public string Order
+        {
+            get
+            {
+                return ((SelectItem)this.configData.GetDynamic("orderrequestedfromsite")).Value;
+            }
+        }
+
+        private Uri SiteUri
+        {
+            get
+            {
+                return new Uri(this.SiteLink);
+            }
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+            var releases = await PerformQuery(new TorznabQuery());
+
+            await ConfigureIfOK(string.Empty, releases.Any(), () =>
+                throw new Exception("Could not find releases from this URL"));
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            try
+            {
+                try
+                {
+                    await this.ConfigureDDoSGuardCookie();
+                }
+                catch (Exception ex)
+                {
+                    this.logger.Log(LogLevel.Warn, ex, $"Exception while configuring DDoS Guard cookie. Attempting search without. (Exception: {ex})");
+                }
+
+                // Get specified categories. If none were specified, use all available.
+                List<string> searchCategories = this.MapTorznabCapsToTrackers(query);
+                if (searchCategories.Count == 0)
+                {
+                    searchCategories = this.GetAllTrackerCategories();
+                }
+
+                // Prepare the search query
+                NameValueCollection queryParameters = new NameValueCollection
+                {
+                    { "page", "search" },
+                    { "id", string.Join(",", searchCategories) },
+                    { "group", "0" }, // No group
+                    { "q", query.SearchTerm ?? string.Empty },
+                    { "s", this.SortBy },
+                    { "o", this.Order }
+                };
+
+                // Make search request
+                Uri searchUri = new Uri(this.SiteUri, "?" + queryParameters.GetQueryString());
```

Comment:
Convert to interpolated strings and remove `this`:

`$"{SiteUri}?{queryParameters.GetQueryString()}"`